### PR TITLE
Example Jobs: fix permission/shebangs and add a multi suite example

### PR DESCRIPTION
**Pull request by clebergnu**
Simples fixes for permission/shebangs, and introduction of a multi suite example based on configs.

### CODE REVIEW

**Review by willianrampazzo — APPROVED** [review]

**Comment by willianrampazzo** [comment]
BTW, isort is failing!

File: examples/jobs/multiple_suites_from_config.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+
+import sys
+
+from avocado.core.job import Job
+from avocado.core.suite import TestSuite
+
+JOB_CONFIG = {'run.test_runner': 'nrunner'}
+
+ORDERLY_CONFIG = {'run.test_runner': 'nrunner',
```
Comment (beraldoleal) [inline examples/jobs/multiple_suites_from_config.py:10]: Maybe we could omit the runner here.

File: examples/jobs/multiple_suites_from_config.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+                  'run.references': ['/bin/true', '/bin/true', '/bin/last'],
+                  'nrunner.max_parallel_tasks': 1}
+
+RANDOM_CONFIG = {'run.test_runner': 'nrunner',
```
Comment (beraldoleal) [inline examples/jobs/multiple_suites_from_config.py:14]: and here.

File: examples/jobs/multiple_suites_from_config.py (hunk 1 above)
Comment (clebergnu) [reply examples/jobs/multiple_suites_from_config.py:10]: > Maybe we could omit the runner here.

Actually, no, the job level runner selection is not being respected here.  And that's kind of the reason for this example.  I'm opening an issue about a bug here.

**Comment by clebergnu** [comment]
> BTW, isort is failing!

Thanks for the heads up, it's now fixed!

File: examples/jobs/multiple_suites_from_config.py (hunk 1 above)
Comment (beraldoleal) [reply examples/jobs/multiple_suites_from_config.py:10]: I did some tests before commenting this and I could not find any "runner' string in test-results log. But maybe we are not logging per test config dict and that is the reason I could not find.

**Review by beraldoleal — APPROVED** [review]

**Pull request merged**